### PR TITLE
Awesome books: a complete website with navigation

### PR DESCRIPTION
**Pull request by VanessaAoki**
#### For Day 4:
- [x] We implemented a navigation bar, with 3 content sections:
   1. Book list;
   2. Add new book form;
   3. Contact info;
- [x] We implemented luxon library to add current date/time to our app.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,25 +1,103 @@
...
+      <h2 class="h2-contact">Contact information</h2>
+      <p>Do you have any questions or just want to say "hello"?</p>
+      <p>You can reach out to us!</p>
+      <h3>👩🏼‍💻 Vanessa Aoki</h3>
+
+      <ul>
+        <li>
+          GitHub:
+          <a href="https://github.com/VanessaAoki" target="_blank">
+            @VanessaAoki
```
Comment (abruzy) [inline index.html:52]: your design should follow this [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m4_full_website.md#project-requirements)

Here's a screenshot of how your design looks like now

Screenshot 2021-08-12 at 21 21 23


PS:

I was surprised to see the above design, I had to checkout to different branches, I was thinking you might have made a mistake or so, but after several checking, I still couldn't find anything close to the screenshot of the design you provided in the readme file. I also saw that you're comparing the navigation branch to the development branch which makes me think I'm on the right branch, maybe you didn't push some updates or so. 

I must say this, I was really wow when I saw the screenshot of the design in the readme but on cloning the project and seeing the above, I was not so happy

Please I don't know what might have gone wrong from your end but please update this and resubmit for review 😄

**Review by abruzy — CHANGES_REQUESTED** [review]
Hi @VanessaAoki & @enionsouza 👋🏽, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

I must really commend your readme, this is one of the awesome readme's I've seen since I started reviewing, you guys keep up with the good work and keep rocking 🚀 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (VanessaAoki) [reply index.html:52]: Hello @abruzy,

First of all, thanks so much for the review!
Turned out we just had an additional "/" when linking our stylesheet, so our CSS was not linked at all.
It should be all fixed now!
If you prefer, here is the [live demo link](https://rawcdn.githack.com/VanessaAoki/AwesomeBooks/3ca746caa2332a3bc69bb2846545a09298e8045f/index.html).

Comment (abruzy) [reply index.html:52]: Oppps! Okay, great work fixing that 🚀 

I really love your design 😄 

Submit again for re-review

**Review by sinansevgi — APPROVED** [review]
# Approved :heavy_check_mark: 

Hi @VanessaAoki and @enionsouza , 

I really liked your design :1st_place_medal: 
Font and color combination looks really good.
Your project is complete!  
Congratulations! 🎉

You can merge this pull request and continue with your journey :wave: 


Happy Hacking! :computer: 

-----

You can reply to this conversation if you have any questions or need elaboration. :wave: 
Sinan Sevgi


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**